python/ray/experimental/client/api.py

```diff
@@ -131,10 +166,136 @@ def call_remote(self, instance: "ClientStub", *args, **kwargs):
     def close(self) -> None:
         return self.worker.close()
 
+    def kill(self, actor: "ClientActorHandle", *, no_restart=True):
+        return self.worker.terminate_actor(actor, no_restart)
+
+    def cancel(self, obj: "ClientObjectRef", *, force=False, recursive=True):
+        return self.worker.terminate_task(obj, force, recursive)
+
+    # Various metadata methods for the client that are defined in the protocol.
+    def nodes(self):
+        """Get a list of the nodes in the cluster (for debugging only).
+
+        Returns:
+            Information about the Ray clients in the cluster.
+        """
+        return self.worker.get_cluster_info(
+            ray_client_pb2.ClusterInfoType.NODES)
+
+    def actors(self, actor_id: Optional[str] = None):
+        """Fetch actor info for one or more actor IDs (for debugging only).
+
+        Args:
+            actor_id: A hex string of the actor ID to fetch information about. If
+                this is None, then all actor information is fetched.
+
+        Returns:
+            Information about the actors.
+        """
+        client_id = None
+        if actor_id is not None:
+            client_id = bytes.fromhex(actor_id)
+
+        return self.worker.get_cluster_info(
+            ray_client_pb2.ClusterInfoType.ACTORS, client_id=client_id)
+
+    def objects(self, object_ref: Optional[str] = None):
```

Comment:
Can we not expose this in the public API? Users always try to use this incorrectly; I believe it's been removed from the documentation.

Reply:
I didn't have the history on that; it is in the API so I figured it'd be nice to cover. Removal is easy enough.